src/store/Workspaces/index.ts

```diff
@@ -333,7 +333,11 @@ export const actionCreators: ActionCreators = {
       return workspace;
     } catch (e) {
       dispatch({ type: 'RECEIVE_ERROR' });
-      throw new Error('Failed to create a new workspace from the devfile: \n' + e);
+      if (e.response?.data?.message) {
+        // e.response.data.message is set unless the request failed. In which case e.message will hold the error code 
+        throw new Error('Failed to create a new workspace from the devfile: \n' + e.response.data.message);
+      }
+      throw new Error('Failed to create a new workspace from the devfile: \n' + e.message);
```

Comment:
In case of 401, it's going to be displayed as Object [Object].
I'll try to reproduce it.
See the login how we handle factory flow on the existing dashboard https://github.com/eclipse/che-dashboard/pull/115/files#diff-138afd46ff894721fbaab6b89d68f102fef94df5d58b7fc1bbaa4118afdc289aR466

Reply:
I missed where it's set but as described in the comment, the response code is displayed
![Screenshot_20201215_110855](https://user-images.githubusercontent.com/5887312/102194394-f122c800-3ec5-11eb-936c-84cc259c2258.png)

Isn't it a bug that promise is not caught?
![Screenshot_20201215_110941](https://user-images.githubusercontent.com/5887312/102194572-25968400-3ec6-11eb-9451-53fb23371938.png)

The reliable way to reproduce it:
1. Open Dashboard next.
2. Scale down Che:
```bash
oc patch deployment/che \
        --patch "{\"spec\":{\"replicas\":0}}
```
3. Try to perform the test operation

Reply:
I've tried to fix the promise of not being caught but it seems like it's not that simple as just suppressing the error, there are a lot of other dependant parts so it might be worth another issue